src/components/HomePage/SearchResultsArea/Offer/OfferCardTheme.js

```diff
@@ -0,0 +1,39 @@
+import AppTheme from '../../../../AppTheme';
+
```

Comment:
Same concerns as with `SearchBarTheme`, seems weird... Would probably prefer this be done directly in the component, otherwise it might get confusing (despite the added code clutter)

Reply:
The styles and code inside the same file is a lot confusing and unreadable. Hence the separation in two files. Due to the generation of more files, the folder separation was done, which I think is a good balance (The style file naming is consistent to avoid confusion)

Reply:
Let's roll with it for now and see how it evolves in the future, for now I'm okay with that

My concern was more on smaller components that would just generate way too many files for what it's worth and might as well just have it inline. As said above, time will tell the best approach